### ISSUE TITLE
Flutter error 

### ISSUE THREAD

**Issue by olivier37520** [Clon1998/mobileraker#135]
I found an error, screenshots provided.

Step to reproduce:
1 Open slide panel
2 click on the current printer settings
3 click on the back button on top left 
![Screenshot_2023-03-28-18-56-12-388_com mobileraker android](https://user-images.githubusercontent.com/6954741/228324617-c14459b9-d341-4517-a02a-4ea2eb4423af.jpg)

**Comment by Clon1998** [comment]
Can you provide me with the app version that is shown at the bottom of the app settings page?

**Comment by olivier37520** [comment]
![Screenshot_2023-03-28-22-36-15-988_com mobileraker android](https://user-images.githubusercontent.com/6954741/228361010-e23799b3-17ed-429d-be2a-a81628a4b129.jpg)

**Comment by olivier37520** [comment]
![Screenrecorder-2023-03-28-22-39-15-48](https://user-images.githubusercontent.com/6954741/228366214-8ccdda14-5216-49f3-9b0f-85e6184c487a.gif)

**Comment by Clon1998** [comment]
Hey thanks for the version.
Please make sure you update to 2.2.1 and let me know if the issue is resolved or not

**Comment by olivier37520** [comment]
Same issue with build from source.

**Comment by sakib-shaikh-github** [comment]
I could only understand the PrimaryScrollController, thing you need wrap that around _builder_ so it could get an context to build.

**Comment by Clon1998** [comment]
> I could only understand the PrimaryScrollController, thing you need wrap that around _builder_ so it could get an context to build.

Nope. The bug is caused by the dependency that provides the reordablewrap.

See: https://github.com/hanshengchiu/reorderables/pull/174
